.prettier.json

```diff
@@ -0,0 +1,9 @@
+{
+  "singleQuote": true,
+  "jsxSingleQuote": true,
+  "semi": false,
+  "tabWidth": 2,
+  "bracketSpacing": true,
+  "jsxBracketSameLine": false,
+  "arrowParens": "always"
+}
```

Comment:
why we need 2 prettier config files? `.prettier.json ` and `.prettierrc`

Reply:
done!